synfig-studio/images/CMakeLists.txt

```diff
@@ -272,6 +272,10 @@ endforeach()
 # Synfig Splash Screen
 if(${CMAKE_BUILD_TYPE} MATCHES Debug)
     set(SYNFIG_SPLASH_SCREEN ${CMAKE_CURRENT_BINARY_DIR}/splash_screen_development.sif)
+    file(
+        COPY ${CMAKE_CURRENT_SOURCE_DIR}/logo.sif
+        DESTINATION ${CMAKE_CURRENT_BINARY_DIR}
+    )
```

Comment:
Stumbled on this [StackOverflow answer](https://stackoverflow.com/a/34800667/13138783)
https://github.com/synfig/synfig/pull/1525#issuecomment-653833203

Better and intelligent CMake code to copy:
```cmake
configure_file(logo.sif logo.sif COPYONLY)
```

**input** is automatically mapped to `CMAKE_CURRENT_SOURCE_DIR`
and **output** to `CMAKE_CURRENT_BINARY_DIR`
so no need to explicitly enter the path variables :)

Reply:
`configure_file` is used to work with `*.in` files to substitute variable values referenced in the file content. Yes, copying is an alternate use of `configure_file`, however, `file(COPY ...)` preserves timestamps of input file as well. 

[CMake configure_file](https://cmake.org/cmake/help/v3.0/command/configure_file.html)
> Copies a file <input> to file <output> and substitutes variable values referenced in the file content.

[CMake file](https://cmake.org/cmake/help/v3.0/command/file.html#copy)
> Copying preserves input file timestamps, and optimizes out a file if it exists at the destination with the same timestamp.

You can check one use of `configure_file` [here](https://github.com/synfig/synfig/blob/master/synfig-studio/images/CMakeLists.txt#L276).

Reply:
https://github.com/synfig/synfig/pull/1525#issuecomment-653851114